src/data/faq.json

```diff
@@ -800,71 +800,9 @@
-                                ]
-                            },
-                            {
-                                "title": "Why is my certificate not verifiable or not recognized?",
-                                "anchor": "val_service_result",
-                                "active": false,
-                                "textblock": [
-                                    "<b>Not recognized</b>",
-                                    "If you receive the message 'Certificate not recognized' as the result of the validation check, then the certificate is technically valid but does not meet the requirements of the provider.",
-                                    "This can be due to several reasons. Possible reasons are, for example:",
-                                    "<ul>",
-                                    "<li>An antigen test is older than 48 hours and has therefore lost its validity</li>",
-                                    "<li>The certificate was not issued by a certified test center</li>",
-                                    "</ul>",
-                                    "<b>Certificate not verifiable</b>",
-                                    "If you receive the message 'Certificate not verifiable' as the result of the validation check, this may be for technical reasons. In this case, your certificate may be technically outdated or there may be a lack of data within the certificate that is needed for verification.",
-                                    "In this case you can contact the issuer of your certificate to obtain a new certificate.",
-                                    "<b>Successful exam</b>",
-                                    "The certificate has been successfully verified and meets the requirements of the provider."
+                                    "Companies must comply with legal requirements to protect the health of employees and customers. In the pandemic, they therefore need reliable access management for business premises, factory halls, aircraft, trams, hotels and restaurants or events for instance. They need to be able to verify the authenticity of Digital COVID Certificates (DCCs) quickly and easily. This is served by a Validation Service.  The Federal Ministry of Health (Bundesministerium für Gesundheit, BMG) has decided to enable several validation services after appropriate verification."

```

Comment:
Not true if the project is on hold:

"The Federal Ministry of Health (Bundesministerium für Gesundheit, BMG) has decided to enable several validation services after appropriate verification."